cinema/views.py

```diff
@@ -0,0 +1,38 @@
+from django.shortcuts import render
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework import status
```

Comment:
Add a blank line between different groups of import. Check imports in the project so they follow PEP-8